lib/Net/Async/Blockchain/BTC.pm

```diff
@@ -145,168 +88,15 @@ L<Ryu::Source>
 
 =cut
 
-sub subscribe {
+async sub subscribe {
```

Comment:
question : why you need to make this sub async?

Reply:
it's a shared function we overwrite it here, so it's async version in other packages like Etehreum